__tests__/app.test.js

```diff
@@ -79,14 +79,70 @@ describe("NC-News", () => {
         });
     });
   });
+  describe("GET /api/articles/:article_id", () => {
+    test("It should respond with an article object by article_id", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article).toBeInstanceOf(Object);
+          expect(body.article.article_id).toBe(1);
+        });
+    });
+    test("It should have the correct keys", () => {
+      return request(app)
+        .get("/api/articles/2")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article).toHaveProperty("author", expect.any(String));
+          expect(body.article).toHaveProperty("title", expect.any(String));
+          expect(body.article).toHaveProperty("article_id", expect.any(Number));
+          expect(body.article).toHaveProperty("body", expect.any(String));
+          expect(body.article).toHaveProperty("topic", expect.any(String));
+          expect(body.article).toHaveProperty("created_at", expect.any(String));
+          expect(body.article).toHaveProperty("votes", expect.any(Number));
+          expect(body.article).toHaveProperty(
+            "article_img_url",
+            expect.any(String)
+          );
+        });
+    });
+  });
```

Comment:
These two above tests are for a previous ticket (ticket 5) so shouldn't be included on this ticket